pip_deps/jetson_requirements.txt

```diff
@@ -1,6 +0,0 @@
-numpy==1.16.2
-<<<<<<< HEAD
-=======
-scipy==1.5.4
->>>>>>> main
-odrive==0.5.1
```

Comment:
@cgiger00 @zwgold what should be in this file?

Reply:
I mentioned this in the simulator path points PR, but 99% sure scipy should no longer be in there. @cgiger00 plz confirm

Reply:
yep